pkg/cmd/util/docker/docker.go

```diff
@@ -47,7 +47,7 @@ func (_ *Helper) GetKubeClient(requestTimeout, imagePullProgressDeadline time.Du
 	} else {
 		endpoint = "unix:///var/run/docker.sock"
 	}
-	client := dockertools.ConnectToDockerOrDie(endpoint, requestTimeout, imagePullProgressDeadline)
+	client := dockertools.ConnectToDockerOrDie(endpoint, requestTimeout, imagePullProgressDeadline, false, false)
```

Comment:
@deads2k this is ok, the last two args are used only for fake docker (for testing).